pkgs/by-name/proximity_matcher_webservice/package.nix

```
@@ -0,0 +1,84 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+let
+  py-tlsh = python3Packages.tlsh.overrideAttrs (oa: rec {
+    pname = "py-tlsh";
+    version = "4.7.2";
+
+    src = fetchFromGitHub {
+      owner = "trendmicro";
+      repo = "tlsh";
+      tag = version;
+      hash = "sha256-fmec8E0LblmBpleHRsJPWO7S3cIbBtFVcHsYQJY/Pns=";
+    };
+
+    postConfigure = ''
+      cd ..
+      find py_ext -maxdepth 1 -type f -exec mv -t $PWD {} \;
+      mv -t $PWD py_ext/pypi_package/*
+      rmdir py_ext/pypi_package
+      rmdir py_ext
+    '';
+  });
+in
+python3Packages.buildPythonPackage {
+  pname = "proximity_matcher_webservice";
+  version = "0-unstable-2023-12-23";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "armijnhemel";
+    repo = "proximity_matcher_webservice";
+    rev = "45675b2abab3596b9bc676fc3e6d643c12528850";
+    hash = "sha256-1MKJhAE9twxFOjdSUd+SaX+6hsuGG6o9NJ2IIGvKGos=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
```

See https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters

```suggestion
  build-system = with python3Packages; [
```